implementations/package.json

```diff
@@ -21,7 +25,8 @@
     "lint": "solhint 'contracts/**/*.sol'",
     "test": "truffle test",
     "solc": "sh solc.sh",
-    "publish": "truffle compile --all; npm run test; npm publish"
+    "publish": "truffle compile --all; npm run test; npm run package; npm publish",
+    "package": "node optimize-artifacts.js"
```

Comment:
reduce-abi is better

Reply:
@frozeman I would disagree here, as the abi is one among many entries in the JSON file.
But in the end, it is just a filename 🙂

Reply:
reduce-artifacts

Reply:
Ah the joy of finding good and meaningful file names! 😄  Haha
Done @frozeman ✅